assets/src/blocks/Covers/CoversEditorScript.js

```diff
@@ -97,3 +97,4 @@ export const registerCoversBlock = () => {
   });
 }
 
```

Comment:
I first removed the function call around it, but it's easier if we preserve it and call it so the file is similar enough during diff/merge.